### DISCUSSION TITLE
How to install it for an existing project?

### DISCUSSION THREAD

**Discussion by Sc4r3k1ng** [NVIDIA/cccl#2907] [Q&A]
Because It's my first time installing the library and I don't know how to do it. I would like to install it to my project. I'm using Windows with Code::Blocks IDE and mingw. Could Someone help me install it? I would appriciate that!

**Comment by pauleonix** [comment]
Do you actually need to install it (i.e. the one coming with the CUDA Toolkit is not new enough)? If yes, there is quite a bit of documentation on this topic in the [readme](https://github.com/NVIDIA/cccl?tab=readme-ov-file#github). What exactly is unclear?
